### ISSUE TITLE
-5 error when logging into counterwallet

### ISSUE THREAD

**Issue by robby-dermody** [CounterpartyXCP/counterwallet#767]
```
failoverAPI: Call failed (failed over across all servers). Method: get_chain_txns_status; Last error: JSON-RPC Error:
Type: Server error

Code: -32000

Message: Got back error from server: {u'message': u'Server error', u'code': -32000, u'data': {u'message': u"{'code': -5, 'message': 'No information available about transaction'} (txhash:: 64f7ccd8eb42c2aef4906323f24849ca6fad8a91fe1bb300a160cbc360c7e0f9)", u'args': [u"{'code': -5, 'message': 'No information available about transaction'} (txhash:: 64f7ccd8eb42c2aef4906323f24849ca6fad8a91fe1bb300a160cbc360c7e0f9)"], u'type': u'BackendRPCError'}}
```

Analysis from J-Dog: issue is when you login to counterwallet, it tries to lookup any pending actions…and that action is failing to locate the transaction hash associated with the pending action…. I just logged into counterwallet.coindaddy.io and got 2 errors….. then right-click->inspect element.. then click the “Resources” tab in inspector window, then click and expand Local Storage.. then click on counterwallet.coindaddy.io or wallet.counterwallet.io…Your now looking at a list of data that counterwallet has written to your device…… find any keys that start with “pendingAction”… highlight it, click the X button…. then reload the main counterwallet window and try again(edited)

basically your blowing away the pending actions… so when you login next time you will have no pending actions, and should not try to lookup the bad tx

**Comment by robby-dermody** [comment]
fixed in 
https://github.com/CounterpartyXCP/counterblock/commit/3b987807e7cca31bcb5a7e7e376b5825a2ee6af3
https://github.com/CounterpartyXCP/counterblock/commit/f69b4dbef897adb0ca5bab11f8d567eb1e5f2f2e
https://github.com/CounterpartyXCP/counterparty-lib/commit/1de43589e6d09b95afc73ecd70a83a5c62a3abb7
https://github.com/CounterpartyXCP/counterparty-lib/commit/a0e0c10db4839b7ed6e83afd2b3d8798cb70289d
